index.html

```diff
@@ -46,6 +46,294 @@ <h1 class="primary-text">
+                                <li class="lang"><a href="#">bootstrap</a></li>
+                                <li class="lang"><a href="#">ruby</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div>
+                        <button class="card-btn see-project button" >
+                            see project
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <!-- end of card -->
+            <!-- card -->
+            <div class="grid-item">
+                <div class="card-img">
+                    <img src="images/work/card-image.png" alt="card image">
+                </div>
+                <div class="card-content">
+                    <div class="card-text">
+                        <div class="card-title">
+                            <p>professional art printing data</p>
+                        </div>
+                        <div class="card-desc">
+                            <p>
+                                A daily selection of privately personalized reads; 
+                                no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                        </div>
+                        <div>
+                            <ul class="tags">
+                                <li class="lang"><a href="#">html</a></li>
+                                <li class="lang"><a href="#">bootstrap</a></li>
+                                <li class="lang"><a href="#">ruby</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div>
+                        <button class="card-btn see-project button" >
+                            see project
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <!-- end of card -->
+            
+        </div>
+    </section>
+    <section class="about container">
+        <div class="top-block">
+            <div class="about-text">
+                <div class="about-title">
+                    <p>About me</p>
+                </div>
+                <div class="about-desc">
+                    <p>Hello I’m a software developer! I can help you build 
+                        a product , feature or website Look through some of 
+                        my work and experience! If you like what you see and 
+                        have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                </div>
+                <div>
+                    <button class="about-btn button">get my resume</button>
+                </div>
+            </div>
+            <div>
+                <img class="mobile-abt-img" src="images/about/mobile-about.png" alt="Mobile about image">
+            </div>
```

Comment:
[OPTIONAL] 👍🏽 
- I think it is better if you replace this `<img> `tag with a proper background image in your CSS file because decorations are always backgrounded images, not img tags. 😉